modin/error_message.py

```diff
@@ -33,17 +33,16 @@ def not_implemented(cls, message=""):
         )
 
     @classmethod
-    @logger_decorator("MODIN-ERROR", "ErrorMessage.single_warning", "debug")
     def single_warning(cls, message):
         message_hash = hash(message)
+        get_logger().debug(f"Modin Warning: Single Warning: {message}")
```

Comment:
```suggestion
        get_logger().debug(f"Modin Warning: Warning `{message}` was raised. If it has already been raised, it will be suppressed.")
```